.github/workflows/parachain.yml

```diff
@@ -34,6 +34,8 @@ jobs:
             ${{ runner.os }}-cargo-
       - name: Install Protoc
         uses: arduino/setup-protoc@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
To prevent Github rate limiting errors: https://github.com/arduino/setup-protoc/issues/18